src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -115,21 +88,21 @@ update_shell(unsigned int index, const std::string& flashType,
   if(!flasher.isValid())
```

Comment:
While you are here :-)
```suggestion
  if (!flasher.isValid())
```